src/components/donations/CryptoTaxCalculator.tsx

```
@@ -0,0 +1,154 @@
+                                </div>
+                                <div className="flex justify-between items-center p-3 bg-red-50 dark:bg-red-900/20 rounded-lg border border-red-200 dark:border-red-800">
+                                    <span className="text-red-700 dark:text-red-300 font-medium">Capital Gains Tax Saved (20%)</span>
+                                    <span className="font-bold text-red-600 dark:text-red-400">+${result.estimatedTaxLiability.toLocaleString(undefined, { maximumFractionDigits: 0 })}</span>
+                                </div>
+                                <div className="flex justify-between items-center p-3 bg-blue-50 dark:bg-blue-900/20 rounded-lg border border-blue-200 dark:border-blue-800">
+                                    <span className="text-blue-700 dark:text-blue-300 font-medium">Tax Deduction Value</span>
+                                    <span className="font-bold text-blue-600 dark:text-blue-400">+${result.taxDeductionValue.toLocaleString(undefined, { maximumFractionDigits: 0 })}</span>
+                                </div>
+                            </div>
+
+                            <button className="w-full py-4 bg-green-600 hover:bg-green-700 dark:bg-green-500 dark:hover:bg-green-600 text-white font-bold text-lg rounded-xl shadow-md transition-colors mt-4">
```

ESLint

[error] Insert `</div>⏎⏎`

(prettier/prettier)